src/basho_bench_stats_writer_csv.erl

```diff
@@ -67,6 +71,11 @@ new(Ops, Measurements) ->
 
 terminate({SummaryFile, ErrorsFile}) ->
     ?INFO("module=~s event=stop stats_sink=csv\n", [?MODULE]),
+
+    %% write out run-wide statistics before exiting
+    RunStatsType = basho_bench_config:get(run_statistics_output_format, json),
```

Comment:
Let's switch the default to csv as the rest of our stats default to outputting as csv.